flutter/pubspec.yaml

```diff
@@ -23,6 +23,7 @@ dev_dependencies:
   yaml: ^3.0.0 # needed for version match (code and pubspec)
   pedantic: ^1.10.0
   build_runner: ^1.11.5
+  platform: ^3.0.0
```

Comment:
how much effort to vendor in this dependency instead?

Reply:
It's just a wrapper around Darts Platform class. This library just has it as instance properties where as Darts platform is static. So it's just easier to test. However it doesn't support web so we need to do it ourselfs anyways.

Reply:
dup https://github.com/getsentry/sentry-dart/pull/389/files#r603078130